frontend/src/app/features/in-app-notifications/entry/in-app-notification-entry.component.sass

```diff
@@ -5,7 +5,7 @@ $ian-bg-hover-color: var(--bgColor-neutral-muted)
 $ian-bg-read-color: var(--bgColor-muted)
 $ian-bg-read-hover-color: hsl(from $ian-bg-read-color h s calc(l - 0.07))
 $ian-bg-selected-color: var(--selection-bgColor)
-$ian-bg-selected-hover-color: hsl(from $ian-bg-selected-color calc(h - 5) s l)
+$ian-bg-selected-hover-color: hsl(from var(--selection-bgColor) h calc(s - 30) l)
```

Comment:
The issue is not that the color is not dark enough. Firefox calculates this to `rgba(0, 0, 0, 0)` in the dark mode, and thus making the effect invisible. Are you able to reproduce this issue? I am currently running on version 127.0.1

Reply:
Ok, I updated to 128.0.2 and it seems to be solved in there 🤷

Reply:
Hi, yes mine is 128.0.2 too, and it is ok there.